beangulp/importer.py

```diff
@@ -177,37 +177,44 @@ class ImporterProtocol:
     # you prefer to create your imported transactions with a different flag.
     FLAG = flags.FLAG_OKAY
 
-    def name(self):
+    def name(self) -> str:
         """See Importer class name property."""
         return f"{self.__class__.__module__}.{self.__class__.__name__}"
 
     __str__ = name
 
     def identify(self, file) -> bool:
         """See Importer class identify() method."""
+        # Type error ignore for backwards compatibility - this should be overridden
+        # and implemented in subclasses
+        return None  # type: ignore
```

Comment:
If returning `None` is not allowed, `return None` should be `raise NotImplementedError`. If it is allowed, the annotation is wrong.

Reply:
sounds good, will change that - wasn't sure whether you'd want this sort of breaking change